dask/dataframe/tests/test_dataframe.py

```diff
@@ -744,20 +744,40 @@ def test_index():
 
 
 def test_assign():
```

Comment:
Should we rename this test so that if it fails we know more clearly where the problem lies?